joinfaces-autoconfigure/src/main/java/org/joinfaces/autoconfigure/rewrite/SpringBootBeanNameResolver.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2016-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.joinfaces.autoconfigure.rewrite;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.ocpsoft.logging.Logger;
+import org.ocpsoft.rewrite.el.spi.BeanNameResolver;
+
+import org.springframework.beans.factory.ListableBeanFactory;
+import org.springframework.context.ApplicationContext;
+
+/**
+ * {@link BeanNameResolver} implementation for Spring. Inspired by
+ * https://github.com/ocpsoft/rewrite/blob/master/integration-spring/src/main/java/org/ocpsoft/rewrite/spring/SpringServiceEnricher.java
+ *
+ * @author Marcelo Fernandes
+ */
+public class SpringBootBeanNameResolver implements BeanNameResolver {
```

Comment:
Does `org.ocpsoft.rewrite.spring.SpringServiceLocator` not work?

Reply:
Same answer as above.